### ISSUE TITLE
Retrieve user's friend list

### ISSUE THREAD

**Issue by fleimeris** [hicore/hicore#2]
Is there a way to retrieve a user's friend list ?
If not, could @farzadi implement this function ? I think this function is must to be, because there's no way to message someone unless you save it locally, but I don't think this is very viable to store friend list this way.

**Comment by farzadi** [comment]
Every time you login to the game,  server sends you a list in response.

**Comment by fleimeris** [comment]
Okay, thanks. But at least for me it seems logical to be able to fetch that kind of information anytime, don't you agree too ?

**Comment by farzadi** [comment]
Yes. I should implement this feature. Because users socket ID changes every time. It's on my to-do list and as soon as I can find time I will implement this feature

**Comment by fleimeris** [comment]
Okay, thank you.
